cmd/limactl/du.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/docker/go-units"
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/lima-vm/lima/pkg/store/dirnames"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newDiskUsageCommand() *cobra.Command {
+	diskUsageCommand := &cobra.Command{
+		Use:               "du",
```

Comment:
Bikeshedding: maybe `df`

Reply:
I know that Docker calls it "df", but it doesn't make sense since it doesn't show how much is free/available ?